src/Faker/Provider/fr_FR/Person.php

```diff
@@ -49,12 +49,46 @@ class Person extends \Faker\Provider\Person
         'Marguerite', 'Margot', 'Margaret', 'Maggie', 'Marianne', 'Marie', 'Marine', 'Marthe', 'Martine', 'Maryse',
         'Mathilde', 'Michèle', 'Michelle', 'Michelle', 'Monique', 'Nathalie', 'Nath', 'Nathalie', 'Nicole', 'Noémi', 'Océane',
         'Odette', 'Olivie', 'Patricia', 'Paulette', 'Pauline', 'Pénélope', 'Philippine', 'Renée', 'Sabine', 'Simone', 'Sophie',
-        'Stéphanie', 'Susanne', 'Suzanne', 'Susan', 'Suzanne', 'Sylvie', 'Thérèse', 'Valentine', 'Valérie', 'Véronique',
+        'Stéphanie', 'Susanne', 'Suzanne', 'Susan', 'Sylvia', 'Sylvie', 'Thérèse', 'Valentine', 'Valérie', 'Véronique',
```

Comment:
We do have some name changing in here? Can we undo that so that the recordset will stay equal?

Reply:
I miss-read your comment that you replaced some duplicate firstnames. Looks fine sorry!

Reply:
I found other duplicates (Michelle, Margot, maybe more). Can remove them all so that we don't break existing unit tests several times?